### DISCUSSION TITLE
23.9 "Aegean"

### DISCUSSION THREAD

**Discussion by Nekosis** [Meowix-Linux/Meowix-ISO#4] [Announcements]
<p align="center">
<img src="https://github.com/Meowix-Linux/Meowix-ISO/blob/main/assets/meowix-text.svg?raw=true" width="50%" height="50%">
</p>

<h1 align="center">Meowix Linux 23.9 "Aegean"</h1>

<p align="center">Welcome to version 23.9 of Meowix Linux, proudly presented by the Meowix maintainers.</p>

<p align="center">Please note that Meowix is currently in its early development stages, which may result in occasional instability. We value your participation and encourage you to <a href="https://github.com/Meowix-Linux/Meowix-ISO/issues">report any bugs or issues</a> you encounter.</p>

## Notable Changes

In this release, we are excited to introduce:

- Critical bug fixes from the previous release, ensuring a smoother and more stable user experience.
- Expanded default software selection, providing you with more apps right out of the box.
- BSPWM layout selector via bsp-layout, making it easier than ever to customize your desktop environment.
- Meowix is now considered production-ready after a fresh installation, offering a dependable computing experience from the start.

##

Your support and engagement are vital to the ongoing development of Meowix Linux. Thank you for being part of our community.

We look forward to your feedback and contributions as we continue refining and expanding Meowix Linux.

<hr /><em>This discussion was created from the release <a href='https://github.com/Meowix-Linux/Meowix-ISO/releases/tag/23.9'>23.9 "Aegean"</a>.</em>

**Comment by RIT-entertainment** [comment]
Bit of a bigger file 🤯
